fury/actors/effect_manager.py

```diff
@@ -0,0 +1,431 @@
+        2. Repeat
+        3. MirroredRepeat
+        4. ClampToBorder
+    border_color : tuple (4, ), optional
+        Texture RGBA border color.
+    interpolate : bool, optional
+        Texture interpolation.
+    d_type : str, optional
+        Texture pixel type, "rgb" or "rgba". Default is "rgb" 
+    """
+
+    windowToImageFilter = WindowToImageFilter()
+    windowToImageFilter.SetInput(window)
+    type_dic = {"rgb" : windowToImageFilter.SetInputBufferTypeToRGB, 
+                "rgba" : windowToImageFilter.SetInputBufferTypeToRGBA,
+                "zbuffer" : windowToImageFilter.SetInputBufferTypeToZBuffer}
+    type_dic[d_type.lower()]()
+    windowToImageFilter.Update()
+
+    texture = Texture()
+    texture.SetMipmap(True)
+    texture.SetInputConnection(windowToImageFilter.GetOutputPort())
+    texture.SetBorderColor(*border_color)
+    texture.SetWrap(WRAP_MODE_DIC[wrap_mode.lower()])
+    texture.SetInterpolate(interpolate)
+    texture.MipmapOn()
+    texture.SetBlendingMode(BLENDING_MODE_DIC[blending_mode.lower()])
+
+    target_actor.GetProperty().SetTexture(texture_name, texture)
+
+
+def texture_to_actor(
+        path_to_texture : str,
+        texture_name : str,
+        target_actor : Actor,
+        blending_mode : str = "None",
+        wrap_mode : str = "ClampToBorder",
+        border_color : tuple = (
+            0.0,
+            0.0,
+            0.0,
+            1.0),
+        interpolate : bool = True):
+    """Pass an imported texture to an actor.
+    Parameters
+    ----------
+    path_to_texture : str
+        Texture image path.
+    texture_name : str
+        Name of the texture to be passed to the actor.
+    target_actor : Actor
+        Target actor to receive the texture.
+    blending_mode : str
+        Texture blending mode. The options are:
+        1. None
+        2. Replace
+        3. Modulate
+        4. Add
+        5. AddSigned
+        6. Interpolate
+        7. Subtract
+    wrap_mode : str
+        Texture wrapping mode. The options are:
+        1. ClampToEdge
+        2. Repeat
+        3. MirroredRepeat
+        4. ClampToBorder
+    border_color : tuple (4, )
+        Texture RGBA border color.
+    interpolate : bool
+        Texture interpolation."""
+    
+    texture = Texture()
+
+    colormapArray = load_image(path_to_texture)
+    colormapData = rgb_to_vtk(colormapArray)
+
+    texture.SetInputDataObject(colormapData)
+    texture.SetBorderColor(*border_color)
+    texture.SetWrap(WRAP_MODE_DIC[wrap_mode.lower()])
+    texture.SetInterpolate(interpolate)
+    texture.MipmapOn()
+    texture.SetBlendingMode(BLENDING_MODE_DIC[blending_mode.lower()])
+
+    target_actor.GetProperty().SetTexture(texture_name, texture)
+
+
+def colormap_to_texture(
+        colormap : np.array,
+        texture_name : str,
+        target_actor : Actor,
+        interpolate : bool = True):
+    """Convert a colormap to a texture and pass it to an actor.
+    Parameters
+    ----------
```

Comment:
same as before, break a line between the first docstring line and the Parameters